### DISCUSSION TITLE
How to blink/highlight windows border or cursor

### DISCUSSION THREAD

**Discussion by fjchen7** [kovidgoyal/kitty#6851] [Q&A] [answered]
Sometimes I lost where I am, especially when I move or neighbor window. I would like to make current window or cursor blink for location indicator. How can I do that? Thanks!

**Comment by kovidgoyal** [comment, marked as answer]
Press any key and the cursor will blink if its visible. Or map a key to
the https://sw.kovidgoyal.net/kitty/actions/#action-toggle_layout action
to zoom into the active window.

**Reply by fjchen7** [reply]
However the blinking cursor is not noticeable. If possible I hope the border or the line of cursor can be highlighted for a while when I move to or neighbor windows. Thanks anyway!

**Reply by kovidgoyal** [reply]
map f1 launch --type=overlay sh -c "echo Hello jimzk; sleep 2s"

and press the f1 key whenever you want to see what window you are in
